config.example.yaml

```diff
@@ -101,6 +101,18 @@ templates:
             options:
               apiURI: http://{domain}/w/api.php
 
+      /{module:graphoid}:
+        x-modules:
+          - name: simple_service
```

Comment:
This sets up the internal backend. We could also easily enable storage here; see https://github.com/wikimedia/restbase/blob/master/config.test.yaml#L121-L126 for an example.